.github/workflows/ci.yml

```diff
@@ -164,6 +164,13 @@ jobs:
             run-tests: 'yes'
             PHP_CS_FIXER_IGNORE_ENV: 1
 
+          - operating-system: 'ubuntu-24.04'
+            php-version: '8.4'
+            job-description: 'tests using PHPUnit 12'
```

Comment:
i like the idea,
i would even wonder to merge it with previous step
(not sure if we need tests on 8.4 with PHPUnit 11 and separately 8.4 with PHPUnit 12)

Reply:
Let's keep it, for now, separate. Because with PHPUnit 11 there are 25787 tests and with PHPUnit 12 the number of tests is 24840 (you can see the numbers [here](https://github.com/6b7562617765726c6f73/PHP-CS-Fixer/commit/21c26fbaa0b006a5c9f4c3937da90d402396fb45)).

I will investigate the reason, maybe this is an expected change and we can have only one test run for PHP 8.4.